src/xrGame/string_table.cpp

```diff
@@ -82,21 +85,24 @@ void CStringTable::FillLanguageToken()
 void CStringTable::SetLanguage()
 {
     if (LanguageID != std::numeric_limits<u32>::max())
-        pData->m_sLanguage = languagesToken.at(LanguageID).name;
+        if (languagesToken.size() > LanguageID)
+            pData->m_sLanguage = languagesToken.at(LanguageID).name;
     else
     {
         pData->m_sLanguage = pSettings->r_string("string_table", "language");
-        auto it = std::find_if(languagesToken.begin(), languagesToken.end(), [](const xr_token& token) {
-            return token.name && token.name == pData->m_sLanguage;
-        });
+        auto it = std::find_if(languagesToken.begin(), languagesToken.end(), [](const xr_token& token)
+            {return token.name && token.name == pData->m_sLanguage;});
```

Comment:
Unrelated change with inconsistent code style.
Should be this:
```suggestion
        {
            return token.name && token.name == pData->m_sLanguage;
        });
```